static/js/app.js

```diff
@@ -1032,17 +1034,17 @@ function initEditor() {
       win: "Ctrl-E",
       mac: "Command-E"
     },
-    exec: function(editor) {
+    exec: function (editor) {
       runExplain();
     }
   }]);
 
-  editor.on("change", function() {
+  editor.on("change", function () {
```

Comment:
I know this codebase is old, but i'd prefer if you disable the auto-formatter to avoid unnecessary churn.

Reply:
I've disabled the auto-formatter to prevent any unnecessary churn in the codebase, and I've also reverted the changes made by the auto-formatter.